dev-python/rtimulib/rtimulib-7.2.1.ebuild

```diff
@@ -0,0 +1,27 @@
+# Copyright 1999-2018 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=6
+
+PYTHON_COMPAT=( python2_7 python3_{4,5,6} )
+
+inherit distutils-r1
+
+MY_P="RTIMULib-${PV}"
+
+DESCRIPTION="Python Binding for RTIMULib, a versatile IMU library"
+HOMEPAGE="https://github.com/RPi-Distro/RTIMULib"
+SRC_URI="https://github.com/RPi-Distro/RTIMULib/archive/V${PV}.tar.gz -> ${MY_P}.tar.gz"
+
+LICENSE="MIT"
+SLOT="0"
+KEYWORDS="~arm ~arm64"
+IUSE="test"
+
+DEPEND="dev-python/setuptools[${PYTHON_USEDEP}]"
+
+S="${WORKDIR}/${MY_P}/Linux/python"
+
+python_test() {
+	esetup.py test
```

Comment:
```
>>> Test phase: dev-python/rtimulib-7.2.1
 * python2_7: running distutils-r1_run_phase python_test
python2.7 setup.py test
usage: setup.py [global_opts] cmd1 [cmd1_opts] [cmd2 [cmd2_opts] ...]
   or: setup.py --help [cmd1 cmd2 ...]
   or: setup.py --help-commands
   or: setup.py cmd --help

error: invalid command 'test'

Reply:
Removed tests, since these are no real unit tests. These are tests for testing the attached hardware sensors. So it don't make sense. Also added amd64 and x86 since it works in this ARCH too.